src/schemas/json/substreams-manifest-schema.json

```
@@ -0,0 +1,173 @@
+{
+  "name": "Substreams Manifest Schema",
+  "description": "A Schema to support in making Substreams manifest files",
+  "type",: "object",
+  "properties" : {
+    "specVersion" : {
+      "enum" : ["v0.1.0"]
+    },
+    "package" : {
+      "type": "object",
```

```suggestion
      "title": "package",
      "description": "A package",
      "type": "object",
```

Required fix: add `title` and `description` properties ([why `title` is required?](https://github.com/SchemaStore/schemastore/blob/master/editor-features.md))

---

@emiliocramer, this suggestion has not been applied, so marking it as a resolved is not correct. 😄